src/main.rs

```diff
@@ -69,9 +71,10 @@ async fn main() {
         &proxy_data_host,
         hostname,
         backend_host,
+        aruna_api,
         storage_backend.clone(),
         data_handler.clone(),
-        endpoint_id,
+        //endpoint_id,
```

Comment:
Please keep the endpoint_id in this struct as it is needed to uniquely identify the specific endpoint / dataproxy